src/Helpers/AnnotateClassInfo.php

```diff
@@ -39,7 +40,7 @@ public function __construct($className)
      */
     public function getModuleName()
     {
-        $relativePath     = str_replace(BASE_PATH . DIRECTORY_SEPARATOR, '', $this->reflector->getFileName());
+        $relativePath = str_replace(BASE_PATH . DIRECTORY_SEPARATOR, '', $this->reflector->getFileName());
         list($moduleName) = explode(DIRECTORY_SEPARATOR, $relativePath);
```

Comment:
I don't think this will work with modules stored in the vendor folder. Perhaps we should use `ModuleLoader` in framework instead?

Reply:
Probably, yeps, been rewriting a lot of stuff to use ClassInfo etc. as well, Primary target was 'get it working again'

Reply:
I had a thought about this `vendor` bit, and I've come to the conclusion it would be a _very_ bad idea to allow for `vendor/vendorname` modules to be annotated. In the common case, `vendor` is something you definitely do not want to touch, whereas if you're developing a module, you generally do not put it in the vendor folder yourself, only people using your module would

I'd say, do not touch `vendor`, only allow for root modules.

Reply:
I do agree however, I would really like to see this bit cleaned up.

Reply:
That's fine, but module development has moved into the vendor folder already and I expect community module devs will do it too

Reply:
Yeah, but not for active development, right? And in that case, at least for me, I'd have the module in the root. Devving from `vendor` is annoying